ts/components/Pinpad/index.tsx

```diff
@@ -19,9 +19,10 @@ const blurElement = (el: TextInput) => el.blur();
 const current = (ref: React.RefObject<TextInput>) => ref.current;
 
 interface Props {
+  activeColor: string;
+  clearOnInvalid?: boolean;
```

Comment:
it seems that now `PinPad` is used only in one place and `clearOnInvalid` is set to `true`, why we need it to be configurable?

Reply:
I have to check but isn't the Pinpad also used during the Onboarding?

Reply:
that's right!